### DISCUSSION TITLE
IBM Model for watsonx Hackathon 2024.

### DISCUSSION THREAD

**Discussion by MuhammadAwais-32013** [ibm-granite/discussions#13] [Q&A] [answered]
Can we use the IBM Granite model from hugging face and then train it on specific data? We want to get the end point of IBM Granite models and then train on colab or Kaggle with integrating with custom front-end UI, deploy on hugging face then push the code ppt and video intro of a project, hugging face project link then submit that repo link. Will it be allowed?

**Comment by mayank31398** [comment, marked as answer]
@MuhammadAwais-32013 I am not aware of the rules for the hackathon.
I suggest reaching out to the hackathon organizers directly.
This is not the correct place for this question.
